app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
+      session[:user_id] = user.id
+      flash[:success] = 'You have successfully logged in'
+      redirect_to root_path
+    else
+      flash.now[:danger] = 'There is something wrong with your login information'
+      render 'new'
+    end
+  end
```

Comment:
We need to set a new `gen_token` in the cookies and update `remeber_digest` attribute each time a user logs in.
We can do this by invoking the `sign_in` helper method in the application controller(updated version of `create_token_and_save` in `controllers/application_controller` in our case. each time a user is successfully authenticated.